### ISSUE TITLE
Check framerate is consistent in the video metadata

### ISSUE THREAD

**Issue by kp-121** [mI-PIV/app#312]
Use the 'output' video to test. Make sure the framerate matches the manually framerate.

**Comment by jack-elliott** [comment]
The meta data using the native camera app on a Samsung A51 reflects payback speed NOT recording speed. So, the meta data is wrong in the case of slow-motion video on that phone at the very least. @kp-121 not sure if we want to remove this feature then or if there is some clever work-around? Found this for iOS https://stackoverflow.com/questions/28202533/getting-slow-motion-meta-data-from-captured-video-in-ios

**Comment by kp-121** [comment]
It looks like the only way we'll be able to get the capture frame rate with videos is when they are recorded with our app. This is because we save the user-defined frame rate. Otherwise, we'll never be able to know the capture frame rate unless the capture rate is added to the metadata (which looks like it never happens).

I was able to find the correct metadata (capture rate instead of frame rate), however I noticed slow motion Android videos don't keep this information in the metadata.
https://github.com/mI-PIV/app/commit/dd8c9cdd9949dff03d7230766aa1b600995918c1

We can keep this feature because it will default to the playback frame rate, and users recording high frame rates outside of the app can still change the FPS in the parameters section.
